CourseSearchBuilderBlobTrigger/__init__.py

```diff
@@ -0,0 +1,65 @@
+import os
+import logging
+import requests
+import traceback
+
+from datetime import datetime

```

Comment:
datetime not used? Perhaps add a start and end time in the logging to use this though?

Reply:
I will remove it, logs will have timestamp in azure